docs/src/test/scala/docs/http/scaladsl/HttpClientExampleSpec.scala

```diff
@@ -8,7 +8,7 @@ import akka.http.scaladsl.model.HttpRequest
 import akka.http.scaladsl.settings.ClientConnectionSettings
 import akka.http.scaladsl.settings.ConnectionPoolSettings
 import docs.CompileOnlySpec
-import org.scalatest.{ Matchers, WordSpec }
+import org.scalatest.{Matchers, WordSpec}
```

Comment:
This is wrongly formatted and broke the build. This should fix it
```suggestion
import org.scalatest.{ Matchers, WordSpec }
```